### DISCUSSION TITLE
Can I generate a new icon myself in workspace there?

### DISCUSSION THREAD

**Discussion by William150722** [LawnchairLauncher/lawnchair#3410] [Q&A] [answered]
![Screenshot_20230403_122026_Lawnchair 2](https://user-images.githubusercontent.com/111726325/229411264-c735d52a-bed7-42b6-8ffc-2d51efd15233.jpg)

Hi All, in code level, is that possible to generate my owe icon here? REQUEST_CREATE_SHORTCUT

**Comment by SuperDragonXD** [comment, marked as answer]
Hello! I know that this reply is a bit late, but you can change the default workspace items via the files found at [the XML resources directory](https://github.com/LawnchairLauncher/lawnchair/tree/15-dev/lawnchair/res/xml).
